source/common/quic/active_quic_listener.cc

```diff
@@ -65,6 +65,20 @@ ActiveQuicListener::ActiveQuicListener(
                                    quic::QuicCryptoServerConfig::ConfigOptions());
   auto alarm_factory =
       std::make_unique<EnvoyQuicAlarmFactory>(dispatcher_, *connection_helper->GetClock());
+  // Set the socket to report incoming ECN.
+  if (receive_ecn && (udp_listener_->localAddress() != nullptr ||
+                      udp_listener_->localAddress()->ip() != nullptr)) {
```

Comment:
Under what circumstances will this address be null? I suspect that's a bug. If so, can we ENVOY_BUG?